### PR TITLE
✏️ Rewording in `docs/en/docs/fastapi-cli.md`

### PR DESCRIPTION
**Pull request by alejsdev**
✏️ Rewording in `docs/en/docs/fastapi-cli.md`

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit f962dd22632ba064aa7f3f1883949cb114fa07e8 at: https://160f7064.fastapitiangolo.pages.dev

File: docs/en/docs/fastapi-cli.md [hunk 1]
```diff
@@ -54,29 +54,25 @@ $ <font color="#4E9A06">fastapi</font> dev <u style="text-decoration-style:singl
...
 
 ## `fastapi dev`
 
-When you run `fastapi dev`, it will run on development mode.
+Running `fastapi dev` initiates development mode.
 
-By default, it will have **auto-reload** enabled, so it will automatically reload the server when you make changes to your code. This is resource intensive and could be less stable than without it, you should only use it for development.
-
-By default it will listen on the IP address `127.0.0.1`, which is the IP for your machine to communicate with itself alone (`localhost`).
+By default, **auto-reload** is enabled, automatically reloading the server when you make changes to your code. This is resource-intensive and could be less stable than when it's disabled, you should only use it for development. It also listens on the IP address `127.0.0.1`, which is the IP for your machine to communicate with itself alone (`localhost`).
```
Comment (ajpinedam) [inline docs/en/docs/fastapi-cli.md:69]:
```suggestion
By default, **auto-reload** is enabled, automatically reloading the server when you make changes to your code. This is resource-intensive and could be less stable than when it's disabled. You should only use it for development. It also listens on the IP address `127.0.0.1`, which is the IP for your machine to communicate with itself alone (`localhost`).
```

**Review by ajpinedam — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 57aa25346e257cb6d8a5ecfc4cbcac1ad4b6ebf9 at: https://8223418a.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
Nice, thanks @alejsdev! :coffee:

**Pull request merged**